docs/_includes/conditional-include-directive.adoc

```diff
@@ -0,0 +1,85 @@
+////
+== Conditional Include Directive
+
+- User manual
+////
+You can include or exclude lines of text in your document from being processing by specifying three conditional include directives:
+
+* ifdef
+* ifndef
+* ifeval
```

Comment:
Something I love about doing these PRs for you is that I learn much more
than I feel I give. It's fantastic!

On Mon, 18 Jan 2016 at 08:17 Dan Allen <notifications@github.com> wrote:

> In docs/_includes/conditional-include-directive.adoc
> <https://github.com/asciidoctor/asciidoctor.org/pull/478#discussion_r49953487>
> :
>
> > @@ -0,0 +1,85 @@
> > +////
> > +== Conditional Include Directive
> > +
> > +- User manual
> > +////
> > +You can include or exclude lines of text in your document from being processing by specifying three conditional include directives:
> > +
> > +* ifdef
> > +* ifndef
> > +* ifeval
>
> Under this list, add a paragraph briefly explaining what constitutes a
> condition.
>
> These conditions make decisions about whether to processes content based
> on facts.
> The facts are typically the existence and/or value of document-level
> attributes (built-in/implicit), defined in the document or via the API or
> cli).
> For example, you may only want to include (or not include) certain content
> when using the html5 backend.
> In order to do that, you'd check for the existence of the backend-html5
> attribute using a conditional include directive.
> We'll cover specific examples in each section.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor.org/pull/478/files#r49953487>.
>
-- 

Sent from Mobile.